### DISCUSSION TITLE
请问大佬们，115出现这个报错怎么解决

### DISCUSSION THREAD

**Discussion by cMuFengcc** [AlistGo/alist#7273] [Q&A]
开WebDAV 策略，302重定向，不会出现报错，但是视频或者其它资源打不开

failed link: failed get link: Post "https://proa
pi.115.com/app/chrome/downurl?t=172737
4566": read tcp ********:43548-
>***********00:443:read: connection reset
by peer

**Comment by zwfpro** [comment]
同问，用小幸运就没有问题，很奇怪
![屏幕截图 2024-09-30 104908.png](https://www.helloimg.com/i/2024/09/30/66fa1021d5db7.png)
